packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1030,6 +1090,27 @@ export class Combobox
       largestSelectedIndicatorChipWidth,
     });
 
+    if (this.allSelected && this.selectAllEnabled) {
+      this.selectedItems.forEach((item) => {
+        const chipEl = this.referenceEl.querySelector<Chip["el"]>(`#${chipUidPrefix}${item.guid}`);
+        if (chipEl) {
+          this.hideChip(chipEl);
+        }
+      });
+    }
+
+    if (
+      this.selectAllComboboxItemReferenceEl &&
```

Comment:
Could this if statement just use the `this.indeterminate` stateful private property instead of using the element?